python/tk_multi_loader/delegate_publish_history.py

```diff
@@ -95,18 +213,9 @@ def _on_before_paint(self, widget, model_index, style_options):
         except:
             pass
             
-            
         # set the little description bit next to the artist icon
-        if sg_item.get("description") is None:
-            desc_str = "No Description Given"
-        else:
-            desc_str = sg_item.get("description")
-        
-        if sg_item.get("created_by") is None:
-            author_str = "Unspecified User"
-        else:
-            author_str = "%s" % sg_item.get("created_by").get("name")
-
+        desc_str = sg_item.get("description") or "No Description Given"         
+        author_str = sg_item.get("created_by").get("name") or "Unspecified User"
```

Comment:
If the intent is that creater_by should always be there, I think you should use [], otherwise it looks like you might have forgot to add a default value. Plus, if the create_by field is ever missing, the "KeyError: created_by" exception string is more telling than a "NoneType has no attribute get'.